README.md

```diff
@@ -174,8 +174,30 @@ Then the product, as well as an engagement for that product, must be created in
 
 #### DefectDojo configuration in RapiDAST
 
-##### Authentication
-First, RapiDAST needs to be able to authenticate itself to a DefectDojo service. This is a typical configuration:
+There are 2 parts to this:
+- First: choose the exporter and configure an authenticating method
+- Second: choose what data to export
+
+The next 2 chapters will describe the 2 exports, and the 3rd one the data to be exported
+
+
+##### Using Google Cloud Storage export
```

Comment:
Let's put this GCP export section to above in the line 175("#### DefectDojo configuration in RapiDAST"), so that it can be independent of DefectDojo configuration.

So, for example, it would look like:
- hey you can store results in the GCP storage.
- or, you can export it to DefectDojo with the defectdojo config. 
- For DefectYou need to configure "defectDojoExport" this case. 
- note when "defectDojoExport" is configured with GCP config, this additional metadata will be stored together in GCP for further use

Reply:
Since, as currently written in the code & config, as well as the final tarball, the GCS export is really tied to data used by DefectDojo, so I kept the "How to export to GCS" under the more global "How to send data to defectdojo". 
I am okay with splitting the 2 ("sending to GCS" and "sending to DD") in the readme, but it might be confusing, most notably the part where you still might need to configure defectdojo data even though you export to GCS.